### ISSUE TITLE
coco128 training problem

### ISSUE THREAD

**Issue by Zengyf-CVer** [ultralytics/yolov5#5862]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### Bug

@glenn-jocher 
I execute this command,then `labels` is 0.As shown below:
```shell
python train.py --img 640 --batch-size 16 --epochs 1 --data ./data/coco128.yaml --cfg ./models/yolov5s.yaml --weights ./weights/yolov5s.pt
```
![2021-12-03_11-19](https://user-images.githubusercontent.com/41098760/144539367-1ce52f4b-baf5-4060-8924-efc8deb2ae95.png)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@Zengyf-CVer 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. Your error is not reproducible:

<img width="1568" alt="Screenshot 2021-12-03 at 04 33 25" src="https://user-images.githubusercontent.com/26833433/144540752-9cf06e82-3c81-456c-842d-d271c0bf867e.png">

We've created a few short guidelines below to help users provide what we need in order to get started investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by Zengyf-CVer** [comment]
@glenn-jocher 
I see that your cuda version is 11.1. It was normal for me to run on cuda 11.1 before. Now my cuda version is 11.3, and then I ran the instructions on the official website:
```shell
pip3 install torch==1.10.0+cu113 torchvision==0.11.1+cu113 torchaudio==0.10.0+cu113 -f https://download.pytorch.org/whl/cu113/torch_stable.html
```
Then there is a situation where `labels` is 0.I guess the new version of torch 1.10.0+cu113 is not compatible with the code related to `Labels` in your YOLOv5 program.If you have the idea of exploring the latest version of torch 1.10.0+cu113, you can build a cuda11.3 platform on colab, and then test it, because this will affect the adaptation of YOLOv5 to future versions of Cuda and Pytorch.

**Comment by hieuvt29** [comment]
@Zengyf-CVer I got the same error but my problem was missing label files for some images, hope this help

**Comment by Zengyf-CVer** [comment]
@hieuvt29 
I am very clear about what you said, thank you for your answer. I use YOLOv5's default data set coco128, which aims to test the effect on Cuda 11.3.

**Comment by Zengyf-CVer** [comment]
@glenn-jocher 
I solved this problem. The problem was not the Cuda version, but the epochs setting was too small. I set `--epochs` to 10, and then it started to be normal at the 5th epoch.
#### But I am not sure why your first epoch can be displayed normally?
![2021-12-06_16-27](https://user-images.githubusercontent.com/41098760/144813015-8e499574-ca75-4c78-9a2b-b3bc98cd4edf.png)
